modules/cnf-reducing-netqueues-using-pao.adoc

```diff
@@ -0,0 +1,346 @@
+            -vendorID: “0x1af4”
+             deviceID: “0x1000”
+
+      net:
+       userLevelNetworking: true        <3>
+       devices:
+        -interfaceName: “eth*”
+
+      net:
+       userLevelNetworking: true        <4>
+       devices:
+        -interfaceName: “!eno1”
+
+     net:
+       userLevelNetworking: true        <5>
+       devices:
+        -interfaceName: “eth0”
+         vendorID: “0x1af4”
+         deviceID: “0x1000”
+
+    #!
+
+      nodeSelector:
+          node-role.kubernetes.io/worker-cnf: ""
+
+----
+Set the queue count to the reserved CPU count for:
+<1>  All devices.
+<2>  All devices that match any of the defined device identifiers.
+<3>  All devices starting with the interface name `eth`.
+<4>  All devices with an interface named anything other than `eno1`.
+<5>  All devices that have an interface name `eth0`, `vendorID` 0x1af4, and `deviceID` 0x1000.
+
+. Apply the performance profile.
++
+[source,terminal]
+----
+$  oc apply -f <your_profile_name>.yaml
+----
+
+[id="verify-queue-status_{context}"]
+== Verify the queue status
+
+In this section, a number of examples illustrate different performance profiles and how to verify the changes are applied.
+
+.Example 1
+
+In this example, the net queue count is set to the reserved CPU count (2) for _all_ supported devices.
+
+The relevant section from the performance profile is:
+
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+metadata:
+ name: performance
+spec:
+  kind: PerformanceProfile
+  spec:
+   cpu:
+    reserved: 0-1  #total = 2
+    Isolated: 2-8
+   net:
+    userLevelNetworking: true
+    [...]
+----
+
+Display the status of the queues associated with a device using the command:
+[source,terminal]
+----
+$ ethtool -l <device>
+----
+[NOTE]
+====
+Run this command on the node where the performance profile was applied.
+====
+
+Before the profile is applied the queue status is:
+
+[source,terminal]
+----
+# ethtool -l ens4
+Channel parameters for ens4:
+Pre-set maximums:
+RX:         0
+TX:         0
+Other:      0
+Combined:   4
+Current hardware settings:
+RX:         0
+TX:         0
+Other:      0
+Combined:   4
+----
+After the profile is applied the queue status is:
+
+[source,terminal]
+----
+# ethtool -l ens4
+Channel parameters for ens4:
+Pre-set maximums:
+RX:         0
+TX:         0
+Other:      0
+Combined:   4
+Current hardware settings:
+RX:         0
+TX:         0
+Other:      0
+Combined:   2 <1>
+----
+<1> The combined channel shows the total count of reserved CPUs for _all_ supported devices is 2. This matches what is configured in the performance profile.
+
+.Example 2
+
+In this example, the net queue count is set to the reserved CPU count (2) for _all_ supported network devices with a specific `vendorID`.
+
+The relevant section from the performance profile is:
+
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+metadata:
+  name: performance
+spec:
+    kind: PerformanceProfile
```

Comment:
I would check the indentation of your YAML examples. this is inconsistent in that metadata.name has 2 spaces in front, but spec.kind has 4. I think we do normally do just 2.